src/components/TransitionLink.js

```diff
@@ -31,20 +32,21 @@ const TransitionLink = ({
           activeClassName={activeClassName}
           partiallyActive={partiallyActive}
           onClick={event => {
-            triggerTransition({
-              event,
-              to,
-              exit,
-              entry,
-              trigger,
-              replace,
-              linkState: state,
-              ...context
-            });
-
+            if (shouldNavigate(event)) {
+              triggerTransition({
+                event,
+                to,
+                exit,
+                entry,
+                trigger,
+                replace,
+                linkState: state,
+                ...context
+              });
+            };
             if (typeof onClick === "function") {
               onClick(event);
-            }
+            };
```

Comment:
I wasn't entirely sure what to do here:
```js
if (typeof onClick === "function") {
  onClick(event);
};
```
I am assuming this is just forwarding a custom onClick event? I decided to leave it outside of the context of `shouldNavigate()`, but it maybe it needs to be in there as well.

Reply:
I agree that this should be outside of shouldNavigate. If an engineer wants their onClick to not fire on cmd+click, they can add a check for that in their onClick function.

Reply:
Actually, just pushed a commit where we pass down wether navigation is happening or not to the onClick, that way the engineer implementing this doesn't need to do the same check we're already doing.